nntrainer/layers/lstm.h

```diff
@@ -150,6 +150,7 @@ class LSTMLayer : public Layer {
   Tensor c_prev;
 
   std::shared_ptr<Var_Grad> mem_cell;
+  std::shared_ptr<Var_Grad> fgio;
```

Comment:
add some description? also for my understanding, this stores the output of the intermediate gates in the LSTM layer, right?

Reply:
You are right. Added. Thanks.